build_node/build_node_builder.py

```diff
@@ -264,6 +264,8 @@ def __request_task(self):
         supported_arches = [self.__config.base_arch]
         if self.__config.base_arch == 'x86_64':
             supported_arches.append('i686')
+        if self.__config.build_src:
```

Comment:
Question, why do we need such `build_src` config option? Don't we always support building srpm? Why not just `supported_arches = [self.__config.base_arch, 'src']`?

Reply:
@Korulag suggested to leave it up to user, because we have s390x nodes and there are too few workers to use them for sources building

Reply:
ok, makes sense, thanks for the clarification